### DISCUSSION TITLE
How to report planned maintenance windows?

### DISCUSSION THREAD

**Discussion by andersnordh** [searxng/searx-instances#614] [Q&A]
Hi, anyone knows how to report a planned maintenance window so that uptime reported on the https://searx.space is not affected?

Have a nice day,
Anders

**Comment by unixfox** [comment]
We can't "pause" the uptime of your instance. You have to tell us if you are doing a long downtime. That's it.

**Reply by Aadniz** [reply]
How long would be defined as *long*?

**Reply by unixfox** [reply]
> How long would be defined as *long*?

It's not about time but about telling us that you care about your instance.

An example is if your instance is below 90% but we never see a message about long downtime then we will definitively remove your instance.

The main point is to keep us updated if you have a downtime for a long time. Like 3H is not considered as "long".

Uptime of 90% is 3d 26m of downtime in a month

**Reply by andersnordh** [reply]
Thanks, good explanation, great.
The reason for my initiial post was that I have had 14.5 hours fibre outage during the last month, hopefully they have it sorted now ...
Have a good one,
Anders
